doc/scripting_tr2_tr3.asc

```diff
@@ -178,60 +179,62 @@ struct SequenceInfo
 
 enum ScriptOpcodes
 {
-    OP_PICTURE = 0,            // ?
-    OP_LISTSTART = 1,          // ?
-    OP_LISTEND = 2,            // ?
-    OP_STARTFMV = 3,           // Play FMV, operand is FMV ID
-    OP_STARTLEVEL = 4,         // Play (interactive) level, operand is level ID
-    OP_STARTCINE = 5,          // Play Cutscene, operand is cutscene ID
-    OP_LEVELCOMPLETE = 6,      // Do level-completion display (no operands)
-    OP_STARTDEMO = 7,          // Play demo level: operand is level ID
-    OP_JUMPTOSEQUENCE = 8,     // Jumps to specified sequence?
-    OP_ENDSEQUENCE = 9,        // End of sequence (no operands)
-    OP_SETTRACK = 10,          // Play soundtrack: operand is soundtrack ID (it precedes opcodes of associated levels)
-    OP_SUNSETENABLED = 11,     // Bartoli's Hideous
-    OP_LOADINGPIC = 12,        // Chapter screen: operand is chapter ID
-    OP_DEADLYWATER = 13,       // Temple of Xian? Kills Lara when she touches water?
-    OP_REMOVEWEAPONS = 14,     // Lose your weapons (no operands)
-    OP_GAMECOMPLETE = 15,      // End of game (no operands)
-    OP_CUTANGLE = 16,          // Associated with cutscenes; a viewpoint control? (one operand?)
-    OP_NOFLOOR = 17,           // For Floating Islands/Thames Wharf? (one operand?)
-    OP_ADDTOINVENTORY = 18,    // Give item; operand is item type (see below)
-    OP_LARASTARTANIMATION = 19,// Item-type 12 state to start level in: operand is state number
-    OP_NUMSECRETS = 20,        // Number of secrets (overrides engine's hardcoded count of them?): operand is that number
-    OP_KILLTOCOMPLETE = 21,    // Probably for Ice Palace boss (no operands)
-    OP_REMOVEAMMO = 22         // Lose your ammo and medipacks? (no operands?)
+    OP_PICTURE = 0,             // Unused. Compiles but does not show in-game. Maybe PSX. Operand is picture ID.
+    OP_PSX_START = 1,           // Unused. Does not compile. Maybe PSX. Operand is track ID.
+    OP_PSX_FMV = 2,             // Unused. Does not compile. Maybe PSX. Operand is FMV ID.
+    OP_FMV = 3,                 // Display Full Motion Video. Operand is FMV ID.
+    OP_LEVEL = 4,               // Start a playable level. Operand is level ID.
+    OP_CINE = 5,                // Display cut scene sequence. Operand is cutscene ID.
+    OP_COMPLETE = 6,            // Display level-completion statistics panel.
+    OP_DEMO = 7,                // Display demo sequence. Operand is demo level ID.
+    OP_PSX_DEMO = 8,            // Unused. Does not compile. Maybe PSX. Operand is demo level ID.
```

Comment:
No, 8 = OP_JUMPTOSEQUENCE  explicitly allows jumping from one sequence to another.